sys/include/hashes/aes128_cmac.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2016 Fundación Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_hashes_aes128_cmac AES128_CMAC
+ * @ingroup     sys_hashes_keyed
+ * @brief       Implementation of the AES128 CMAC hashing function
+
+ * @{
+ *
+ * @file
+ * @brief       AES128_CMAC interface definition
+ *
+ * @author      José Ignacio Alamos <jose.alamos@inria.cl>
```

Comment:
I can do it later too, but just in case you are in the mood: could you use my HAW address? That email doesn't exist anymore :(

Reply:
well, there are many occurrences of that email in the master branch... I can also do it at some point

Reply:
Then I leave that to you :-D